src/components/MetricChart/MetricChart.js

```diff
@@ -119,6 +119,29 @@ const GenericMetricChart = ({ chartConfig, showGrid }) => {
       })
     }
 
+    if (chartConfig.gradient) {
+      const canvasHeight = showGrid ? 200 : 80
+      if (chartInstance.current.options.scales.x.grid.display !== showGrid) {
+        chartInstance.current.options.scales.x.grid.display = showGrid
+        chartInstance.current.options.scales.y.grid.display = showGrid
+
+        chartInstance.current.options.scales.y.display = showGrid
+
+        chartInstance.current.options.scales.x.grid.ticks = true
+        chartInstance.current.options.scales.y.grid.ticks = true
+      }
+      const gradient = ctx.createLinearGradient(0, 0, 0, canvasHeight)
+      gradient.addColorStop(
+        0,
+        hexToRGB(chartConfig?.data?.datasets[0].backgroundColor || '#FFF', 0.7)
+      )
+      gradient.addColorStop(1, hexToRGB(chartConfig?.data?.datasets[0].backgroundColor))
+      chartInstance.current.data.datasets.forEach(dataset => {
+        dataset.backgroundColor = gradient
+      })
+      chartInstance.current.update()
+    }
```

Comment:
why do you need to set gradient via JS you can do it in css

Reply:
i didn't find required solution only with css